pythnet/pythnet_sdk/src/wire.rs

```diff
@@ -385,4 +389,28 @@ mod tests {
         // The deserialized value should be the same as the original.
         assert_eq!(deserialized_update, empty_update);
     }
+
+    // Test if the AccumulatorUpdateData major and minor version increases work as expected
+    #[test]
+    fn test_accumulator_forward_compatibility() {
```

Comment:
Can you also add a new byte to the message and see how it performs with that?

Reply:
In which field do you mean?

Reply:
i mean probably this:
1. create a message
2. serialize it to `data`
3. append some bytes to `data`
4. make sure deserializing the `data` to message struct works